### ISSUE TITLE
qos: Bug when only one direction for an interface exists

### ISSUE THREAD

**Issue by oxo42** [liquidtelecom/splunk-snmpmod#5]
For the qos input

```
interfaces = 38,152,394
```

```
[[ObjectType(ObjectIdentity(ObjectName('1.3.6.1.4.1.9.9.166.1.2.1.1.1.38.2')), Gauge32(1115038995)), 
ObjectType(ObjectIdentity(ObjectName('1.3.6.1.4.1.9.9.166.1.2.1.1.1.152.2')), Gauge32(339866455)), 
ObjectType(ObjectIdentity(ObjectName('1.3.6.1.4.1.9.9.166.1.2.1.1.1.394.1')), Gauge32(694114417))], 
[ObjectType(ObjectIdentity(ObjectName('1.3.6.1.4.1.9.9.166.1.2.1.1.1.39.2')), Gauge32(1063460152)), 
ObjectType(ObjectIdentity(ObjectName('1.3.6.1.4.1.9.9.166.1.2.1.1.1.154.1')), Gauge32(1444555409)), 
ObjectType(ObjectIdentity(ObjectName('1.3.6.1.4.1.9.9.166.1.2.1.1.1.394.2')), Gauge32(1970903313))]
]
```

When 38.1 doesn't exist snmpwalk seems to pick up 39.2 because it's the next in the list.  Change from `walk_oids` to `query_oid`

**Comment by oxo42** [comment]
In qos.py:260 change the `walk_oids` to `get_oids`

Also check that interface directions that can't be found are handled cleanly

**Comment by carlos-agea** [comment]
Bug #5 Fixed with release v2.6.2
